docroot/modules/humsci/hs_capx/src/Overrides/ConfigOverrides.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Drupal\hs_capx\Overrides;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\hs_capx\Capx;
+use Drupal\key\Entity\Key;
+
+/**
+ * Class ConfigOverrides.
+ *
+ * @package Drupal\hs_capx\Overrides
+ */
+class ConfigOverrides implements ConfigFactoryOverrideInterface {
+
+  /**
+   * Config factory service.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * ConfigOverrides constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   Config factory service.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory) {
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * Override the path to the key for the encryption profile.
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+    if (in_array('migrate_plus.migration.hs_capx', $names) || in_array('migrate_plus.migration.hs_capx_images', $names)) {
```

Comment:
Instead of wrapping 99% of the body of the function in an if statement you could simply return the empty array here.